Assets/Colyseus/Room.cs

```diff
@@ -76,9 +79,13 @@ public int id
 			get { return this._id; }
 			set {
 				this._id = value;
-				this.OnJoin.Emit (this, EventArgs.Empty);
-			}
-		}
+#if !WINDOWS_UWP
+                this.OnJoin.Emit (this, EventArgs.Empty);
+#else
+                this.OnJoin.Invoke(this, EventArgs.Empty);
+#endif
```

Comment:
Looks like `Emit` is an alias to `Invoke`. I didn't know that when I first write this. We can use always `Invoke`. Can we get rid of the conditional compilation?